testsuite/features/secondary/buildhost_docker_build_image.feature

```diff
@@ -85,8 +90,10 @@ Feature: Build container images
   Scenario: Rebuild the images
     When I schedule the build of image "suse_simple" with version "Latest_simple" via API calls
     And I schedule the build of image "suse_key" with version "Latest_key-activation1" via API calls
-    And I wait at most 660 seconds until image "suse_key" with version "Latest_key-activation1" is built and inspected successfully via API
-    And I wait at most 660 seconds until image "suse_simple" with version "Latest_simple" is built and inspected successfully via API
+    And I wait at most 600 seconds until image "suse_key" with version "Latest_key-activation1" is built successfully via API
```

Comment:
I would suggest the following order change:
1. suse_simple built
2. suse_key built
3. suse_simple verified
4. suse_key verified

as it might bring less waiting, and it's somehow more logical

Reply:
Mine is just an opinion, but I'm totally fine with this PR.
I would suggest to organize these scenarios in a way that we don't mention different images inside the same scenario.
So, having the action and validation for each image building in isolation.